running GA
OFFSPRING_SIZE is 5
for 1 generations
seed: 1
my workdir = /global/cscratch1/sd/zladd/axonstandardized/playground/gen_alg_GPU
Traceback (most recent call last):
  File "genetic_alg/optimize_parameters_genetic_alg.py", line 4, in <module>
    import hoc_evaluatorGPU_allen_par as hoc_ev
  File "/global/cscratch1/sd/zladd/axonstandardized/playground/gen_alg_GPU/genetic_alg/hoc_evaluatorGPU_allen_par.py", line 26, in <module>
    orig_params = h5py.File(run_volts_path+'params/params_bbp_full_allen_gpu_tune.hdf5', 'r')['orig_full'][0]
  File "/global/homes/z/zladd/.conda/envs/.env/lib/python3.6/site-packages/h5py/_hl/files.py", line 408, in __init__
    swmr=swmr)
  File "/global/homes/z/zladd/.conda/envs/.env/lib/python3.6/site-packages/h5py/_hl/files.py", line 173, in make_fid
    fid = h5f.open(name, flags, fapl=fapl)
  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "h5py/h5f.pyx", line 88, in h5py.h5f.open
OSError: Unable to open file (unable to open file: name = '../run_volts_bbp_full_gpu_tuned/params/params_bbp_full_allen_gpu_tune.hdf5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
srun: error: cgpu05: task 0: Exited with exit code 1
srun: Terminating job step 781753.0
